src/components/Base.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Container } from '@material-ui/core';
+
+import Routes from './Routes';
+import VerificationBanner from './auth/VerificationBanner';
+
+import useStyles from './_jss/Base.jss';
+
+const Base: React.FC = () => {
+  const classes = useStyles();
+
+  return (
+    <>
+      <VerificationBanner />
+      <Container className={classes.container} fixed>
+        <Routes />
+      </Container>
```

Comment:
we could just move our routes into this file - routes shouldn't ever become too complex and, conceptually, would fit in well with a base component

Reply:
Base will have more stuff relating to content layout in the future so gonna keep this out of it for now. We can change it if need be.

Reply:
did I comment this twice? whoop